client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -46,7 +49,37 @@ object DiagramView {
               s"A diagram for ${cfg.title}"
```

Comment:
Here we just want to render the title that has been entered. If you want to keep "A diagram for ..." for the initialisation, please change the Command handler to populate the initial values for the config case class accordingly.